app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  validates :username, presence: true
+  validates :email, presence: true
+  validates :password, presence: true
+end
```

Comment:
- **[OPTIONAL]** We can properly validate the email to have a valid email [format](https://stackoverflow.com/questions/4776907/what-is-the-best-easy-way-to-validate-an-email-address-in-ruby).